src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react';
+
+import {
+  DEFAULT_PROPS,
+  STATS_ICONS,
+  STATS_LABELS,
+  DEFAULT_AVATAR_ICON,
+  FALLBACK_INITIALS,
+} from './ProfileCard.constants';
+import {
+  StyledCard,
+  StyledProfileContainer,
+  StyledAvatar,
+  StyledInfoContainer,
+  StyledName,
+  StyledUsername,
+  StyledRole,
+  StyledStatsContainer,
+  StyledStatItem,
+  StyledStatIcon,
+  StyledStatContent,
+  StyledStatValue,
+  StyledStatLabel,
+} from './ProfileCard.styles';
+
+import type { ProfileCardProps, StatsItemProps } from './ProfileCard.types';
+
+const StatsItem: React.FC<StatsItemProps> = ({ icon, label, value, 'data-testid': dataTestId }) => {
+  const formattedValue = typeof value === 'number' ? value.toLocaleString('en-US') : value;
+
+  return (
+    <StyledStatItem $variant={DEFAULT_PROPS.variant} data-testid={dataTestId}>
```

Comment:
StatsItem uses `DEFAULT_PROPS.variant` for `$variant` (StyledStatItem and StyledStatContent). This means all stat items are styled as the default variant, ignoring the `variant` passed to the parent ProfileCard. This is a functional bug: the `variant` prop is not propagated.

**Sugerencia:**
```typescript
Change StatsItem to accept a `variant` (or `$variant`) prop and forward it to styled children. Example:

const StatsItem: React.FC<StatsItemProps & { $variant: ProfileCardVariant }> = ({ icon, label, value, 'data-testid': dataTestId, $variant }) => (
  <StyledStatItem $variant={$variant} data-testid={dataTestId}>...
    <StyledStatContent $variant={$variant}>...
  </StyledStatItem>
);

Then when rendering: `<StatsItem $variant={variant} ... />` (pass the parent `variant`).
```